tests/test_sqs/test_sqs.py

```diff
@@ -313,6 +313,29 @@ def test_message_retention_period():
     assert len(messages) == 0
 
 
+@mock_sqs
+def test_queue_retention_period():
```

Comment:
Do we need this test? From what I can see, the correct behaviour is already coveredin the test above (`test_message_retention_period`)

Reply:
Ah, ignore me - I just realized why this test is useful.